packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "@material/elevation/mixins";
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "@material/typography/mixins";
+
+$mdc-toolbar-height: 64px;
+$mdc-toolbar-mobile-height: 56px;
+$mdc-toolbar-padding: 24px 16px;
+$mdc-toolbar-mobile-padding: 16px;
+
+.mdc-toolbar {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: $mdc-toolbar-height;
+  padding: $mdc-toolbar-padding;
+  box-sizing: border-box;
+
+  @media (max-width: 599px) {
```

Comment:
perhaps we should create a variable for this `599px` breakpoint, and put a `TODO` to refactor it out once #23 is implemented?